src/content/insertable-streams/webgpu/js/webpgu-worker.js

```diff
@@ -0,0 +1,80 @@
+
+'use strict';
+
+class WebGPUWorker {
+    constructor() {
+        this.worker_ = null;
+        this.worker2_ = null;
+        this.offscreen_ = null;
+        this.screencanvas_ = null;
+        this.offscreen2_ = null;
+        this.screencanvas2_ = null;
+    }
+
+    async init() {
+        console.log('[WebGPUWorker] Initializing WebGPUWorkers.');
+        this.screencanvas_ = document.createElement('canvas');
+        const screenCanvas = this.screencanvas_;
+        document.getElementById('outputVideo').append(screenCanvas);
+        const offScreen = screenCanvas.transferControlToOffscreen();
+        this.offscreen_ = offScreen;
+
+        this.screencanvas2_ = document.createElement('canvas');
+        const screenCanvas2 = this.screencanvas2_;
+        document.getElementById('gumOutputVideo').append(screenCanvas2);
+        const offScreen2 = screenCanvas2.transferControlToOffscreen();
+        this.offscreen2_ = offScreen2;
```

Comment:
I think maybe I'm missing the point of this sample. If I understand correctly, we're rendering each video feed into a separate canvas, and letting the browser do compositing of the canvases. I don't see a reason this would be better than just using video elements directly, and it seems much more complicated.

I think I'll need to understand the context a bit better before I can review the worker part.